apps/server/src/database/prisma/repositories/prisma-locations-repository.ts

```diff
@@ -58,6 +58,24 @@ export class PrismaLocationsRepository implements ILocationsRepository {
           }
     }
 
+    async findByName(locationName: string): Promise<Location | null> {
+        try {
+            const prismaLocation = await prisma.location.findFirst({
```

Comment:
Prefer to use `findUnique` instead of `findFirst` , as the former is more efficient and semantic.